website/docs/r/multisite_site_association.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_multisite_site_association"
+sidebar_current: "docs-vcd-resource-multisite-site-association"
+description: |-
+  Provides a resource to create, read, and modify a VMware Cloud Director site association with the current site.
+---
+
+# vcd\_multisite\_site\_association
+
+Provides a resource to create, read, and modify a VMware Cloud Director site association.
+
+~> Note: this resource requires System Administrator privileges
+
+Supported in provider *v3.13+*
+
+## Example Usage 1
+
+Creating a site association using an association data file.
+
+To create an association in the current site (`site1`), we need a file containing the other site information, which its
+administrator will have created using the data source `vcd_multisite_site_data`.
+
+```hcl
+resource "vcd_multisite_site_association" "site1-site2" {
+  association_data_file = "site2.xml"
+}
+```
+
+## Example Usage 2
+
+Creating a site association using association data from some other source.
+
+```hcl
+resource "vcd_multisite_site_association" "site1-site2" {
+  association_data = var.remote_data
+}
+```
+Here, `var.remote_data` contains the same XML that was generated by `vcd_multisite_site_data` in the other site.
```

Comment:
Would be good to add an example on how to generate the `var.remote_data` in the other site. It's not entirely clear.

Reply:
Added more explanation, pointing to an existing example

Reply:
Moreover, the whole workflow is explained in detail in the guide (referred in each data source and resource doc file)

Reply:
Thanks. A link would be fine too.

Please amend the same paragraph in a different file:
https://github.com/vmware/terraform-provider-vcd/pull/1260/files#diff-12665ae1a5087806e2ded0f0476e02b5d9988a790d99fddc701318c6a8420badR39

Reply:
done